csrc/scheduler/multi_matmul.cpp

```diff
@@ -239,10 +670,81 @@ class MultipleMatmulScheduler {
   // Currently the support is for a, b, c and d as fusion inputs/outputs
   //  aka. no prolog fusion yet.
   void defineOperandCaches() {
-    cacheOperandsToSmem(as_, acw_smems_, params_.supported_vec_size.a);
-    addSetsForCacheReads(acw_smems_, acrs_);
+    LoadStoreOpType load_op = params_.async_gmem_load_operands
+        ? LoadStoreOpType::CpAsync
+        : LoadStoreOpType::Set;
+
+    auto cacheOperandsToSmem = [&](const std::vector<TensorView*>& operands,
```

Comment:
We changed this to a member function in https://github.com/NVIDIA/Fuser/pull/2719, but it is changed back as a lambda here. Is this intentional?

Reply:
Oh thanks for noticing. I just botched the merge. I will update it to reflect the change on main.